Automattic/wp-calypso
# Add akismet enterprise benefits to checkout

## Proposed Changes

* Add similar product benefits to Akismet Enterprise checkout as the other Akismet products (only differing with API limit)
* For Enterprise only, add % saved buying a yearly plan instead of a monthly plan

## Testing Instructions

1. Checkout branch and start local environment or click on the Calypso live link
2. Go to `/checkout/akismet/ak_ep2m_yearly_1`
3. Make sure you see the product benefits including the `17% price reduction for yearly term`
![image](https://github.com/Automattic/wp-calypso/assets/65001528/e0fb3623-a36e-4ecc-b85f-da6bbc0a343f)
4. Do the same for `/checkout/akismet/ak_ep350k_yearly_1`
![image](https://github.com/Automattic/wp-calypso/assets/65001528/f43ed9e8-0ce4-499d-851b-08581e38ca68)

## Pre-merge Checklist
